src/content/docs/support/faq.mdx

```diff
@@ -110,7 +110,7 @@ error: failed retrieving file 'telegram-desktop-2.5.1-1-x86_64.pkg.tar.zst' from
 This means that your local database is outdated therefore the package you're asking for can't be downloaded. Run the following command to update your local database:
```

Comment:
Can you update this to match the below command then?

Reply:
I reworded it to reflect what the command does :)